README.md

```diff
@@ -79,11 +79,40 @@ Use the `LangChain` CLI to bootstrap a `LangServe` project quickly.
 
 To use the langchain CLI make sure that you have a recent version of `langchain-cli`
 installed. You can install it with `pip install -U langchain-cli`.
+## Setup
+**Note**: We use `poetry` for dependency management. Please follow poetry [doc](https://python-poetry.org/docs/) to learn more about it.
 
+### 1. Create new app using langchain cli command
 ```sh
-langchain app new ../path/to/directory
+langchain app new my-app
 ```
 
+### 2. Activate `poetry` shell. Every command after step 2 should be executed within `poetry` shell.
+```sh
+poetry shell
```

Comment:
Let's not recommend poetry shell here. Instead have folks run 

`poetry run [command]` (e.g,. `poetry run pytest`)

Reply:
Done